src/JsonSchema/Metadata/Property/Factory/SchemaPropertyMetadataFactory.php

```diff
@@ -509,6 +513,13 @@ private function getLegacyClassType(?string $className, bool $nullable, ?bool $r
             ];
         }
 
+        if (class_exists(\BcMath\Number::class) && is_a($className, \BcMath\Number::class, true)) {
```

Comment:
```suggestion
        if (is_a($className, \BcMath\Number::class, true)) {
```

IIRC, the call to `is_a()` will return false if the class doesn't exist.